src/RobotInterface/CMakeLists.txt

```diff
@@ -2,10 +2,19 @@
 # This software may be modified and distributed under the terms of the
 # GNU Lesser General Public License v2.1 or any later version.
 
-if (FRAMEWORK_COMPILE_RobotInterface)
-
 set(H_PREFIX include/BipedalLocomotion/RobotInterface)
 
+if (FRAMEWORK_COMPILE_RobotInterface AND FRAMEWORK_COMPILE_Perception)
```

Comment:
I think we discussed having separate libraries, right? Otherwise, we are back to the old discussion of needing ``opencv`` every time we need to communicate with the robot.

Reply:
I added the if..else logic for this purpose. 
Only if opencv and pcl are found the relevant sources are added to the `RobotInterface` library.
Or if you think it would be cleaner to create separate libraries, I could do it.

Reply:
Ah, I did not notice the else branch. To be honest I do not like that some targets may have the public headers to be variable. In this way, it would be difficult for the user library to understand if the ``ICameraBridge`` is available or not. I would rather go for separate libraries.

Reply:
Understood. I will add a library name `PerceptionInterface`

Reply:
Done in https://github.com/dic-iit/bipedal-locomotion-framework/pull/165/commits/d74a670c6c0b81a3c773ce911910fb04784f8a0d